README.md

```diff
@@ -3,18 +3,281 @@
 
-
-It's a golang dubbo library used by kitex (https://github.com/cloudwego/kitex).
+Golang dubbo library used by kitex (https://github.com/cloudwego/kitex) for **kitex-dubbo interoperability**.
 
 
 ## Feature List
 
+### Type Mapping
 
+|    thrift type     |   golang type    | hessian2 type |       java type        |
+|:------------------:|:----------------:|:-------------:|:----------------------:|
+|        bool        |       bool       |    boolean    |   java.lang.Boolean    |
+|        byte        |       int8       |      int      |     java.lang.Byte     |
+|        i16         |      int16       |      int      |    java.lang.Short     |
+|        i32         |      int32       |      int      |   java.lang.Integer    |
+|        i64         |      int64       |     long      |     java.lang.Long     |
+|       double       |     float64      |    double     |    java.lang.Double    |
+|       string       |      string      |    string     |    java.lang.String    |
+|       binary       |      []byte      |    binary     |         byte[]         |
+|    list\<bool>     |      []bool      |     list      |     List\<Boolean>     |
+|     list\<i32>     |     []int32      |     list      |     List\<Integer>     |
+|     list\<i64>     |     []int64      |     list      |      List\<Long>       |
+|   list\<double>    |    []float64     |     list      |     List\<Double>      |
+|   list\<string>    |     []string     |     list      |     List\<String>      |
+|  map\<bool, bool>  |  map[bool]bool   |      map      | Map\<Boolean, Boolean> |
+|  map\<bool, i32>   |  map[bool]int32  |      map      | Map\<Boolean, Integer> |
+|  map\<bool, i64>   |  map[bool]int64  |      map      |  Map\<Boolean, Long>   |
+| map\<bool, double> | map[bool]float64 |      map      | Map\<Boolean, Double>  |
+| map\<bool, string> | map[bool]string  |      map      | Map\<Boolean, String>  |
 
-## reference
+**Important notes**:
+1. The list of map types is not exhaustive and includes only tested cases.
+Please do not use keys and values with **i8**, **i16** and **binary**.
+2. Currently only the thrift type and java type mappings documented in the table are supported.
+More mappings(e.g. **bool** <-> **boolean**) would be supported in the future.
+Please see [issue](https://github.com/kitex-contrib/codec-dubbo/issues/46).
+
+### Kitex-Dubbo Interoperability
+
+1. **kitex -> dubbo**  
+Writing **api.thrift** based on existing **dubbo Interface API** and [**Type Mapping Table**](#type-mapping). Then using
+latest kitex command tool and thriftgo to generate stub code.
+2. **dubbo -> kitex**  
+Writing dubbo client code based on existing **api.thrift** and [**Type Mapping Table**](#type-mapping).
+
+## Getting Started
+
+[**Concrete sample**](https://github.com/kitex-contrib/codec-dubbo/tree/main/samples//helloworld/).
+
+### Prerequisites
+
+```shell
+# compile kitex cmd tool
+mkdir ~/dubbo && cd ~/dubbo
+git clone https://github.com/Lvnszn/kitex.git
```

Comment:
It should be the official repository. Wait before @Lvnszn 's commit to be merged first.

Reply:
Could be simplied as `go install github.com/cloudwego/kitex/tool/cmd/kitex@xxx`